app/views/users/new.html.erb

```diff
@@ -2,25 +2,6 @@
 
 <h1>Add New User</h1>
 
-<div class="row">
-  <div class="span6 offset3">
-    <%= form_for(@user) do |f| %>
+<%= render 'form' %>
 
-      <%= render 'shared/error_messages' %>
-
-      <%= f.label :name %>
-      <%= f.text_field :name %>
-
-      <%= f.label :login %>
-      <%= f.text_field :login %>
-
-      <%= f.label :password %>
-      <%= f.password_field :password %>
-
-      <%= f.label :password_confirmation, "Confirmation" %>
-      <%= f.password_field :password_confirmation %>
-
-      <%= f.submit "Create account", class: "btn btn-large btn-primary" %>
-    <% end %>
-  </div>
-</div>
+<%= link_to 'Back', groups_path %>
```

Comment:
link is to group page instead of user page

Reply:
Hi @bjkeller,
Thank you for feedback.